src/components/Graph/Graph.tsx

```diff
@@ -111,6 +111,7 @@ const Graph: FC<GraphProps> = memo(({ pathway, interactive = true, expandCurrent
   );
 
   const toggleExpanded = useCallback((key: string) => {
+    if (key === 'Start') return;
```

Comment:
I think this line is causing some weirdness with the expand/unexpand logic when switching back & forth between nodes. It's definitely a super corner case to switch back & forth between Start and anything else, but if you have Node B open, click Start, then click back to Node B, it closes B instead of keeping it open. (It does work correctly for any other pair of nodes that don't include Start). If there's an easy fix for that let's make it here, otherwise I'd suggest reverting this line and making a new task to keep the start node from expanding.